packages/polling-controller/src/PollingController.ts

```diff
@@ -49,19 +59,26 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
      * Starts polling for a networkClientId
      *
      * @param networkClientId - The networkClientId to start polling for
+     * @param options - The options used to group the polling events
      * @returns void
      */
-    startPollingByNetworkClientId(networkClientId: NetworkClientId) {
+    startPollingByNetworkClientId(
+      networkClientId: NetworkClientId,
+      options: object = {},
```

Comment:
Since we end up stringifying these options, maybe instead of `object`, we should use the `Json` type?